src/handlebars/support.handlebars

```diff
@@ -6,8 +6,53 @@
   <h1 class="margin-bottom large-title">Support</h1>
 
   <div class="align-left support">
-    <div class="anchor" onmouseover="document.getElementById('anchor1').style.visibility='visible'" onmouseout="document.getElementById('anchor1').style.visibility='hidden'"><a href="#quality" class="anchor">
+
+    <div class="anchor" onmouseover="document.getElementById('anchor1').style.visibility='visible'" onmouseout="document.getElementById('anchor1').style.visibility='hidden'"><a href="#community-support" class="anchor">
       <img src="dist/assets/anchor.png" id="anchor1" name="anchor" alt="anchor icon"></a>
+      <h2 class="bold"><a id="support-levels">Community Support</a></h2>
+    </div>
+    <div class="margin-bottom">
+      <p>
+        Read <a href="https://medium.com/@javachampions/java-is-still-free-c02aef8c9e04">Java Is Still Free</a>
+        for some background information. We are prepared to stand behind our release quality binaries, so each
```

Comment:
consider "... for some background information about Java support options."

Reply:
Fixed